InterviewQuestions/Chapters/CustomLinkedLists/CustomLinkedList.cs

```diff
@@ -1,39 +1,73 @@
-﻿namespace InterviewQuestions.Chapters.CustomLinkedLists
-{
-    public class CustomLinkedList
-    {
-        public class Node
-        {
-            public int data;
-            public Node? next;
-            public Node(int data)
-            {
-                this.data = data;
-                this.next = null;
-            }
-        }
-        public Node? head;
-
-        public CustomLinkedList()
-        {
-            head = null;
-        }
-        public void add(int newElement)
-        {
-            Node newNode = new Node(newElement);
-            if (head == null)
-            {
-                head = newNode;
-            }
-            else
-            {
-                Node currentNode = head;
-                while (currentNode.next != null)
-                {
-                    currentNode = currentNode.next;
-                }
-                currentNode.next = newNode;
-            }
-        }
-    }
-}
+﻿using System.Diagnostics;
+using System.Security;
+
+namespace InterviewQuestions.Chapters.CustomLinkedLists
+{
+    public class CustomLinkedList
+    {
+        public class Node
+        {
+            public int data;
+            public Node? next;
+            public Node(int data)
+            {
+                this.data = data;
+                this.next = null;
+            }
+        }
+        public Node? head;
+
+        public CustomLinkedList()
+        {
+            head = null;
+        }
+        public void add(int newElement)
+        {
+            Node newNode = new Node(newElement);
+            if (head == null)
+            {
+                head = newNode;
+            }
+            else
+            {
+                Node currentNode = head;
+                while (currentNode.next != null)
+                {
+                    currentNode = currentNode.next;
+                }
+                currentNode.next = newNode;
+            }
+        }
+        public int getLenghtLinkedList()

```

Comment:
Just getLenght() would be better

Reply:
Done. You can look here bc628d64f6458686147b1cd8f98a8020470fc47d